### ISSUE TITLE
[rx-51] gps3 driver segfault

### ISSUE THREAD

**Issue by craftyguy** [postmarketOS/pmbootstrap#575]
Reported by @pavelmachek in chat, when running gps3 [1], he experiences a segfault.

gps3 is required to provide an interface between the n900's gps hardware and the gpsd daemon.

When this issue is resolved, we should also package gps3 in pmOS (and upstream it to Alpine)

1. http://luke.dashjr.org/tmp/code/gps3.c

**Comment by ollieparanoid** [comment]
Regarding `gps3.c`, I recommend that we fork that and put it somewhere in GitHub, `/tmp/code/` doesn't look suitable for maintenance.

**Comment by craftyguy** [comment]
Yes, my intention is to fork it, but since this is pretty far on my list the only thing I could do for now is include a pointer to it so I don't forget where it came from later :)

**Comment by craftyguy** [comment]
There is no license information at the link I posted, so I have contacted the individual who posted it. I suppose there's a change this is not being distributed with a compatible license, in which case we'll be stuck. I will report back if/when I get a response from them.

**Comment by luke-jr** [comment]
(Confirming)

**Comment by pavelmachek** [comment]
I started packaging gps3. https://github.com/postmarketOS/gps-nokia-n900

**Comment by pavelmachek** [comment]
And I have it packaged. ... would placing it in pmbootstrap/aports/device/gps-nokia-n900/APKBUILD be ok?

**Comment by ollieparanoid** [comment]
yep, that makes sense since it's only for the n900
